.github/workflows/ci.yml

```diff
@@ -0,0 +1,33 @@
+name: Python
+
+on:
+  [push, pull_request]
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+
+    runs-on: ubuntu-20.04 # latest version doesn't have python 3.5
```

Comment:
Is this comment needed? Python 3.5 and 3.6 are EOL: https://www.activestate.com/solutions/python-version-support/

Reply:
good point, I can drop that comment